ui/src/utils/patterns.js

```diff
@@ -8,7 +8,7 @@ const
   rgba = /^rgba\(((0|[1-9][\d]?|1[\d]{0,2}|2[\d]?|2[0-4][\d]|25[0-5]),){2}(0|[1-9][\d]?|1[\d]{0,2}|2[\d]?|2[0-4][\d]|25[0-5]),(0|0\.[0-9]+[1-9]|0\.[1-9]+|1)\)$/
 
 export const testPattern = {
-  date: v => /^([0-9]{4}[\/]((0[13-9]|1[012])[\/](0[1-9]|[12][0-9]|30)|(0[13578]|1[02])[\/]31|02[\/](0[1-9]|1[0-9]|2[0-8]))|([0-9]{2}(([2468][048]|[02468][48])|[13579][26])|([13579][26]|[02468][048]|0[0-9]|1[0-6])00)[\/]02[\/]29)$/.test(v),
+  date: v => /^-?([0-9]+[\/]((0[13-9]|1[012])[\/](0[1-9]|[12][0-9]|30)|(0[13578]|1[02])[\/]31|02[\/](0[1-9]|1[0-9]|2[0-8]))|([0-9]{2}(([2468][048]|[02468][48])|[13579][26])|([13579][26]|[02468][048]|0[0-9]|1[0-6])00)[\/]02[\/]29)$/.test(v),
```

Comment:
`-0/01/01` works according to your regex, as does `-9996/02/29` and the absolutely insane:

`-2000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000/01/01`

Reply:
tested at https://regex101.com

Reply:
@nothingismagick I am happy to change the regex back to only allowing a year value with a max of 4 digits if that is how we want to proceed.  The original regex statement allowed crazy long years however, which I why I currently have included that in my regex.

Reply:
The origional regex statement:
`-?[\d]+\/[0-1]\d\/[0-3]\d`